osd/BadMachinePoolTaint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "summary": "Action required: Bad MachinePool Configuration",
+    "description": "Your cluster requires you to take action. SRE has observed that there have been unsupported changes made to one of the cluster MachinePools. ${REASON}. Please revert the changes.",
```

Comment:
could you add what are the consequences if they keep the unsupported change ?

Reply:
I added a very specific message now, as this should only be send due to an MNMO alert - if this changes in the future, I guess the notification has to be made more generic again.